src/Object/Contact.php

```diff
@@ -746,22 +721,21 @@ public static function getPostsFromUrl($contact_url)
 
 		// There are no posts with "uid = 0" with connector networks
 		// This speeds up the query a lot
-		$r = q("SELECT `network`, `id` AS `author-id`, `contact-type` FROM `contact`
-		WHERE `contact`.`nurl` = '%s' AND `contact`.`uid` = 0", dbesc(normalise_link($contact_url)));
+		$r = dba::select('contact', array('network', 'id AS author-id', 'contact-type'), array('nurl' => normalise_link($contact_url), 'uid' => 0), array('limit' => 1));
```

Comment:
This is some query that I guess will not work. Although I could add some code to bypass these statements, I wouldn't like to do so, since it would open a potential security hole.